.github/stale.yml

```diff
@@ -0,0 +1,24 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - bug
+  - bug - failure to compile
+  - bug - ice
+  - bug - incorrect behaviour
+  - bug - memory leak
+  - bug - miscompile
+  - bug - regression
+  - bug - segfault
+  - feature_request
```

Comment:
should probably add the `performance*`, `enhancement` and `task` labels too? I think the `stale`ness will most likely apply to issue which are not successful in triage due to lack of information.

Any thoughts on the `question` label? Am wondering if this bot could close questions after e.g. a week of inactivity and also mention that `discourse` as the place to keep the Q&A corpus?

Reply:
I'll add the labels yes.

But I don't think this bot supports these custom kinds of messages yet.